executor/tests/agents/test_dify_agent.py

```
@@ -194,84 +215,90 @@ def test_call_dify_api_error_response(self, mock_post, task_data):
+    def test_call_dify_api_http_error(self, mock_post, task_data, mock_emitter):
         """Test Dify API call with HTTP error"""
         # Mock HTTP error
         mock_response = MagicMock()
         mock_response.status_code = 404
         mock_response.raise_for_status.side_effect = Exception("404 Not Found")
         mock_post.return_value = mock_response
 
-        agent = DifyAgent(task_data)
+        agent = DifyAgent(task_data, mock_emitter)
 
         with pytest.raises(Exception):
```

Ruff (0.15.0)

[warning] Do not assert blind exception: `Exception`

(B017)